roaster-zephyr/roaster-zephyr-common/src/main/java/hu/icellmobilsoft/roaster/zephyr/common/client/RestZephyrService.java

```diff
@@ -47,6 +48,11 @@
 @ApplicationScoped
 public class RestZephyrService {
 
+    /**
+     * Maximum depth of the test case structure
+     */
+    public static final int MAX_TEST_CASE_DEPTH = 2;
```

Comment:
A doc-ban 3 van. Értem, hogy a `>` jel miatt ez itt 2-t jelent, de ne használjunk itt más értékeket, lehet `>=`-t is használni.
Valamint nem érdemes ezt konfigba kiemelni?

Reply:
Emberi 3-asra átírva, default paraméterezve.